.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "eslint.workingDirectories": [
+    "./packages/shared",
+    "./packages/webapp",
+    "./packages/extension",
+  ]
```

Comment:
I don't get this error, would love to jump on a quick call to see whats the issue, because it should work as is.

Reply:
@sshanzel are you using the workspace that is provided in `.vscode/apps.code-workspace`, you can open that file manually from "File -> Open Workspace from file" or `code .vscode/apps.code-workspace`

Workspace is configured to include above directories, can you try?

Reply:
Right, let me try and put the config in that direction 👀

Reply:
Worked perfectly fine after that 🥳 🙏